inc/components/video/component.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "irving/video",
+  "alias": "irving/text",
+  "description": "Render an HTML video embed responsively.",
+  "config": {
+    "aspect_ratio": {
+      "default": "16:9",
+      "description": "Required. Video ratio as a CSS property, ex. \"padding-bottom: `{aspect_ratio}%\". Alias options include `square`, `1;1`, `4:3`, `16:9`.",
+      "type": "string"
+    },
+    "video_url": {
+      "default": "",
+      "description": "Required. URL of the video. Currently supports YouTube and Vimeo.",
```

Comment:
Wouldn't basically anything listed in the providers list [here](https://developer.wordpress.org/reference/classes/wp_oembed/) work? (In which case, I think we should drop calling out certain providers by name here).

Reply:
From what I can tell, the [React oEmbed Container package](https://github.com/humanmade/react-oembed-container) only supports some stuff, not everything that WP does. To avoid confusion/being misleading, I called out YouTube and Vimeo specifically..